src/mono/wasm/build/WasmApp.LocalBuild.props

```diff
@@ -22,7 +22,7 @@
   <Import Project="$(MSBuildThisFileDirectory)WasmApp.props" />
 
   <PropertyGroup>
-    <_NetCoreAppToolCurrent>net7.0</_NetCoreAppToolCurrent>
+    <_NetCoreAppToolCurrent>net6.0</_NetCoreAppToolCurrent>
```

Comment:
Rename this property to `_TargetFrameworkForNETCoreTasks` to be similar to what `src/tasks/Directory.Build.props` has, and so it's clear what this version should be.

Reply:
@radical Why isn't this in `/Directory.Build.props` near other version props (or indeed defined in terms of one of those props)? https://github.com/dotnet/runtime/blob/main/Directory.Build.props#L43-L65

Reply:
The `*localBuild*` files get used outside the tree, like when building tests on helix.